### ISSUE TITLE
Error when opening spyder

### ISSUE THREAD

**Issue by Murmele** [spyder-ide/spyder#5439]
## Description

**What steps will reproduce the problem?**

1.  Graphics backend Gtk3
2. Open Spyder

**What is the expected output? What do you see instead?**
Expect no error message

**Please provide any additional information below**
After the error I have to reopen spyder a few times and then it works fine. It could not connect to the python kernel (In console: connection to kernel...)

  File "/usr/lib/python3.6/site-packages/spyder/plugins/__init__.py", line 511, in <lambda>
    toggled=lambda checked: self.toggle_view(checked),
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 695, in toggle_view
    self.create_new_client(give_focus=False)
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 974, in create_new_client
    self.connect_client_to_kernel(client)
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 995, in connect_client_to_kernel
    stderr_file)
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 1383, in create_kernel_manager_and_kernel_client
    stderr = codecs.open(stderr_file, 'w', encoding='utf-8')
  File "/usr/lib/python3.6/codecs.py", line 895, in open
    file = builtins.open(filename, mode, buffering)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/spyder-martin/kernel-4d681953b37a.stderr'

## Version and main components

* Spyder Version: 3.2.0 
* Python Version: 3.6.2
* Qt Versions: 5.9.1, PyQt5 5.9 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  None (NOK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.4 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.0 (OK)
psutil >=0.3     :  5.3.1 (OK)
nbconvert >=4.0  :  5.1.1 (OK)
sympy >=0.7.3    :  None (NOK)
cython >=0.21    :  None (NOK)
qtconsole >=4.2.0:  4.3.0 (OK)
IPython >=4.0    :  5.3.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by Jeroendevr** [comment]
I see you don't use the latest Spyder version. It is possible to upgrade to 3.2.3. By upgrading you distribution or `pip install --upgrade spyder`

**Comment by ccordoba12** [comment]
@Jeroendevr is right. I think this problem was fixed in 3.2.3, or it'll be fixed in 3.2.4 (to be released in a couple of days).
